test/src/test/java/org/corfudb/runtime/view/AbstractViewTest.java

```diff
@@ -106,6 +106,7 @@ public AbstractViewTest(boolean followBackpointers) {
         CorfuRuntime.overrideGetRouterFunction = this::getRouterFunction;
         runtime = CorfuRuntime.fromParameters(CorfuRuntimeParameters.builder()
                 .followBackpointersEnabled(followBackpointers)
+                .garbageCollectionEnable(false)
```

Comment:
You should also check AbstractIT and universe framework to disable gc in their runtime as well.